manim/mobject/numbers.py

```diff
@@ -158,8 +157,8 @@ class Variable(VMobject):
     ----------
     var : Union[:class:`int`, :class:`float`]
         The python variable you need to keep track of and display.
-    label : Union[:class:`str`, :class:`~.Tex`, :class:`~.MathTex`, :class:`Text`]
-        The label for your variable, for example `x = ...`. To use math mode, for e.g.
+    label : Union[:class:`str`, :class:`~.Tex`, :class:`~.MathTex`, :class:`Text`, :class:`TexSymbol`, :class:`SingleStringMathTex`, :class:`MathTexFromPresetString`]
```

Comment:
From a user point of view, I find it confusing to list all subclasses of `SingleStringMathTex`, but this is maybe a discussion for another day. Anyhow: the references to the classes are not set correctly (you can check: they are not rendered as links right now).

```suggestion
    label : Union[:class:`str`, :class:`~.Tex`, :class:`~.MathTex`, :class:`~.Text`, :class:`~.TexSymbol`, :class:`~.SingleStringMathTex`, :class:`~.MathTexFromPresetString`]
```